src/lottery/routes/globalState.js

```diff
@@ -2,10 +2,9 @@ const express = require("express");
 
 const router = express.Router();
 const globalStateHandlers = require("../services/globalState");
-
-// 라우터 접근 시 로그인 필요
-router.use(require("../../middlewares/auth"));
+const auth = require("../../middlewares/auth");
 
 router.get("/", globalStateHandlers.getUserGlobalStateHandler);
+router.post("/create", auth, globalStateHandlers.createUserGlobalStateHandler);
```

Comment:
`auth` 미들웨어를 라우터 내부에 넣는 형태가 아닌 로그인이 필요한 API가 하단, 로그인이 필요 없는 API가 하단에 위치하도록 코드 구성하면 좋을 것 같습니다.

```suggestion

router.get("/", globalStateHandlers.getUserGlobalStateHandler);

// 라우터 접근 시 로그인 필요
router.use(require("../../middlewares/auth"));
router.post("/create", globalStateHandlers.createUserGlobalStateHandler);
```